examples/camera/first_person_view_model.rs

```diff
@@ -0,0 +1,250 @@
+                        fov: 90.0_f32.to_radians(),
+                        ..default()
+                    }
+                    .into(),
+                    ..default()
+                },
+                WorldModelCamera,
+            ));
+            parent.spawn((
+                Name::new("View Model Camera"),
+                Camera3dBundle {
+                    camera: Camera {
+                        // Bump the order to render on top of the world model.
+                        order: 1,
+                        ..default()
+                    },
+                    projection: PerspectiveProjection {
+                        fov: 70.0_f32.to_radians(),
+                        ..default()
+                    }
+                    .into(),
+                    ..default()
+                },
+                // Only render objects on layer 1.
+                RenderLayers::layer(1),
+            ));
+
+            parent.spawn((
+                Name::new("Arm"),
+                MaterialMeshBundle {
+                    mesh: arm,
+                    material: arm_material,
+                    transform: Transform::from_xyz(0.2, -0.1, -0.25),
+                    ..default()
+                },
+                // Ensure the arm is only rendered by the view model camera.
+                RenderLayers::layer(1),
+                // The arms is free-floating, so shadows would look weird.
+                NotShadowCaster,
+            ));
+        });
+}
+
+fn spawn_world_model(
+    mut commands: Commands,
+    mut meshes: ResMut<Assets<Mesh>>,
+    mut materials: ResMut<Assets<StandardMaterial>>,
+) {
+    let floor = meshes.add(Plane3d::new(Vec3::Y, Vec2::splat(10.0)));
+    let cube = meshes.add(Cuboid::new(2.0, 0.5, 1.0));
+    let material = materials.add(Color::WHITE);
+    // The world model camera will render the floor and the cube.
+    // Assigning no `RenderLayers` component defaults to layer 0.
+
+    commands.spawn(MaterialMeshBundle {
+        mesh: floor,
+        material: material.clone(),
+        ..default()
+    });
+
+    commands.spawn(MaterialMeshBundle {
+        mesh: cube.clone(),
+        material: material.clone(),
+        transform: Transform::from_xyz(0.0, 0.25, -3.0),
+        ..default()
+    });
+
+    commands.spawn(MaterialMeshBundle {
+        mesh: cube,
+        material,
+        transform: Transform::from_xyz(0.75, 1.75, 0.0),
+        ..default()
+    });
+}
+
+fn spawn_lights(mut commands: Commands) {
+    commands.spawn((
+        PointLightBundle {
+            point_light: PointLight {
+                color: Color::from(tailwind::ROSE_300),
+                shadows_enabled: true,
+                ..default()
+            },
+            transform: Transform::from_xyz(-2.0, 4.0, -0.75),
+            ..default()
+        },
+        // The light source illuminates both the view model and the world model.
+        RenderLayers::from_layers(&[0, 1]),
```

Comment:
I think I've seen `RenderLayers:all()`

Reply:
I've left it like this intentionally to be explicit about the involved layers. I can change it if you disagree.

Reply:
I changed my opinion: [`a7e69b6` (#13828)](https://github.com/bevyengine/bevy/pull/13828/commits/a7e69b68bd9f37e989b810ac97bdb21c535422c9)
I tried out your suggestion and it looks better this way. Thanks!